### DISCUSSION TITLE
Question about multi domain and TLS

### DISCUSSION THREAD

**Discussion by Geobert** [static-web-server/static-web-server#324] [Q&A] [answered]
Hi,

I have domain New and domain Old. I’m interested in serving a blog from domain New, and send a permanent redirect on domain Old.
It seems doable using redirect, but my question is about certificates: I have 2, one for each domains.

Is there a way to feed them in SWS?

**Comment by joseluisq** [comment, marked as answer]
If you want to use TLS, you can not feed multiple certificates to the same "SWS" instance as an Nginx server does for example.
However, if possible for you to use a multi-domain SSL certificate like [Subject Alternative Name (SAN)](https://support.dnsimple.com/articles/what-is-ssl-san/), then you could use the SWS [Virtual Hosting](https://static-web-server.net/features/virtual-hosting/) feature that will allow you to serve your files from different root directories depending on the `Host` header.

Otherwise, if SAN is not possible for you then you have to use multiple instances of SWS and redirect from one to another.

**Comment by Geobert** [comment]
Thanks for your answer!
